src/PKSim.Core/Services/StatisticalDataCalculator.cs

```diff
@@ -1,22 +1,29 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Maths;
 using OSPSuite.Utility.Extensions;
 using PKSim.Core.Model;
 using PKSim.Core.Model.PopulationAnalyses;
-using OSPSuite.Core.Extensions;
-using OSPSuite.Core.Maths;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static PKSim.Core.Services.StatisticalDataCalculator;
 
 namespace PKSim.Core.Services
 {
    public interface IStatisticalDataCalculator
    {
-      IEnumerable<float[]> StatisticalDataFor(FloatMatrix sortedResults, StatisticalAggregation statisticalAggregation);
+      IEnumerable<float[]> StatisticalDataFor(FloatMatrix sortedResults, StatisticalAggregation statisticalAggregation, DeviationModes deviationMode = DeviationModes.Range);
```

Comment:
I would not make this mode optional. The results are DIFFERENT based on the mode and it should be  required